.github/workflows/browserslist-db-update.yml

```
@@ -8,8 +8,32 @@ on:
   workflow_dispatch:
 
 jobs:
+  close-existing-pr:
+    name: Close existing PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if branch exists
+        id: check-branch
+        continue-on-error: true
+        uses: fjogeleit/http-request-action@v1
+        with:
+          url: 'https://api.github.com/repos/${{ github.repository }}/git/ref/heads/update/browserslist-db'
```

Not essential, but it would be nice to use a top-level `env.branch-name` or similar rather than the inline branch name here and in the `update-browserslist-database` job.

```suggestion
          url: 'https://api.github.com/repos/${{ github.repository }}/git/ref/heads/${{ env.branch-name }}'
```

---

Updated, thank you!